landoui/landoapi.py

```diff
@@ -119,6 +131,22 @@ def from_environment(cls, token: Optional[str] = None) -> LandoAPI:
         )
 
 
+class TreestatusAPI(API):
```

Comment:
A parameter passed to the class would also have been an acceptable approach, but I don't see anything wrong with a separate class in this instance. I think I'm going to leave this as-is, it seems like more work than it's worth to change now, especially with the added context that we will be removing all of this code in Django-Lando anyways.

Reply:
This approach adds more complexity and technical debt than is needed. I.e., separate "service names", as well as separate classes, which takes away from the fact that the two classes are both "Lando API" wrappers wrapping the same API. It may be easier to visualize my point by picturing what this will look like in the unified Lando project -- there is no concept of `LandoAPI` and `TreestatusAPI` -- everything is in the same project and is directly called from within the project itself.